PR_REVIEW.md

```diff
@@ -0,0 +1,29 @@
+## Summary
```

Comment:
if this is verbatim from prebid.js maybe just link it instead of copy/paste.

Reply:
It is modified file specifically for plugin.